### PR TITLE
metiq: filter out negative video latencies

### PR DESCRIPTION
**Pull request by JohanBlome**
A negative video latency indicates that we are going oast the beep period. A warning is printed for every hit that triggers. If many values are filtered out the beep perod needs to be increased.

### CODE REVIEW

File: src/verify.py [hunk 1]
```diff
@@ -0,0 +1,533 @@
+#!/usr/bin/env python3
+
+"""Generate and run test content"""
```
Comment (chemag) [inline src/verify.py]: Not sure I get it. Is this a unittest, or a verification script that you run on the post-analysis?

File: src/verify.py [hunk 2]
```diff
@@ -0,0 +1,533 @@
...
+    )
+
+
+OLD_FRAME = None
+BLACK_FRAME = None
+
+
+def write_frame(rawstream, frame, frame_num, freeze_frames, black_frames):
+    global OLD_FRAME
+    global BLACK_FRAME
```
Comment (chemag) [inline src/verify.py]: in general, global variables are messy (code is harder to read, and may create unexpected code flows). Can they be replaced?

File: src/verify.py (hunk 1 above)
Comment (chemag) [reply src/verify.py]: If its a unittest, please make sure the file name includes the words "unittest" (e.g. "verify-unittest.py"), and that you add in the commit message an example of how to run it. Also, you may want to add some prefix with more meaning than just "verify".

Comment (JohanBlome) [reply src/verify.py]: OK. I moved all test related files to a separate fold named "test".
The file is split up into verify_generate.py for the content creation and verify_unittests.py where the tests are much simpler to write:
"
def test10():
    """
    Frozen frames at sync position. No delays and perfect sync.
    """
    audio = 0.0
    video = 0.0
    descr = f"{inspect.currentframe().f_code.co_name}_vd.{video}s.ad.{audio}s"
    # In 60 fps number scheme
    frames = [*range(160, 190)]
    settings = {
        "outfile": f"{descr}.mov",
        "descr": f"{descr}",
        "output_fps": 60,
        "video_delay": video,
        "audio_delay": audio,
        "freeze_frames": frames,
    }

    return run_test(**settings)
"

In verify.py test cases will be automatically populated from the verify_tests.py file and available test can be queried and run separately. Test content can be kept after test run or deleted (default).

File: src/verify.py (hunk 2 above)
Comment (JohanBlome) [reply src/verify.py]: Removed.

**Pull request merged**